scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/so.yaml

```diff
@@ -47,3 +47,57 @@ SO-4_4.4x3.9mm_P2.54mm:
   pitch: 2.54
   num_pins_x: 0
   num_pins_y: 2
+
+Texas_PowerPAD_SO-20-1EP_7.52x12.83mm_P1.27mm:
+  size_source: 'http://www.ti.com/lit/ds/symlink/opa569.pdf'
+  #custom_name_format: Texas_PowerPAD_SO-{pincount:d}-1EP_{size_x:.2g}x{size_y:.2g}mm_P{pitch:.2g}mm
+  body_size_x:
+    minimum: 7.45
+    maximum: 7.59
+  body_size_y:
+    minimum: 12.7
+    maximum: 12.95
+  overall_size_x:
+    minimum: 10.16
+    maximum: 10.65
+  lead_width:
+    minimum: 0.35
+    maximum: 0.51
+  lead_len:
+    minimum: 1.35
+    maximum: 1.53
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 10
+
+  # http://www.ti.com/lit/an/slma004b/slma004b.pdf
+  EP_size_x:
+    nominal: 6.045 # 0.238"
+    tolerance: 0
```

Comment:
give the size of the exposed pad (package side) here and use EP_size_x_overwrite to define the size on the pcb (this way we can reuse the same info for the 3d model sometime in the future)

Reply:
now added both EP_size and _overwrite lines.